docker/podman-docker.csh

```diff
@@ -0,0 +1,10 @@
+# DOCKER_HOST initialization
+
+if ($?DOCKER_HOST) exit
+if ( $uid == 0 ) then
```

Comment:
The sh equivalent is checking the euid.  Should this be looking at $euid instead, assuming csh is always tcsh?

Reply:
I think we go with euid, and then if these are not set because of a shell that does not support it, we will  set the DOCKER_HOST iff the XDG_RUNTIME_DIR is also set.